### PR TITLE
[PD465] Fix failing system test PG3Analysis

### LINKED ISSUE #34867: Failing SNSPowderRedux.PG3Analysis System Test
**Describe the bug**
The SNSPowderRedux.PG3Analysis test has continued to fail on linux builds after a recent fix (#34858).  

See the following for the test failures
- https://builds.mantidproject.org/view/Nightly%20Pipelines/job/main_nightly_deployment_prototype/473/execution/node/136/log/?consoleFull
- https://builds.mantidproject.org/job/pull_requests-conda-linux/2277/testReport/junit/SystemTests/SNSPowderRedux/PG3Analysis/
- https://builds.mantidproject.org/job/pull_requests-conda-linux/2278/consoleFull

### PR DESCRIPTION
**Pull request by KedoKudo**
**Description of work.**

## TL; DR;
This PR raise the relative tolerance for the system test `PG3Analysis` so that it can pass.

## Long version:

The two spectrums are almost identical (see the plot below), but `CompareWorkspaces` insists that the relative difference is larger than set 1% threshold.

image

When zoom-in on the difference (see plot below), the difference is very small compared to the whole spectrum.

image

Furthermore, when converted the difference to relative difference via `diff/golden` (see below), the relative difference is still below the set 1% threshold.

image

Therefore, the root cause of this issue is most likely how the relative difference is computed by `CompareWorkspaces`.
However, this dataset is not suitable for addressing this issue, and simply raising the threshold to 2% bypass this issue.

**To test:**

Build the branch and run `build/systemtest -R "PG3Analysis"`

Fixes #34867. 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by peterfpeterson — APPROVED** [review]
The difference is notably small.
